data_collection/gazette/spiders/pa/pa_parauapebas.py

```diff
@@ -0,0 +1,47 @@
+import datetime as dt
+
+import scrapy
+from dateparser import parse
```

Comment:
Como usamos bastante o `parse` no contexto de raspagem, seria preferível evitar o import direto da função e assim o contexto ficaria explícito usando `dateparser.parse`.
```suggestion
import dateparser
```